client/connector.go

```diff
@@ -83,10 +77,12 @@ type connector struct {
 	// use columnID instead of name since column name can be reused
 	// table names can be reused as well, deleting and adding a new table
 	// will anyway requires job restart
+	// enum mapping is only needed for enum columns with auto expand disabled for enum translation
```

Comment:
I think we should have Column definition, and enum mapping should go into each column level. Can keep here first if the change is too big